app/controllers/friendships_controller.rb

```diff
@@ -28,9 +29,11 @@ def update
   end
 
   def destroy
-    friendship = Friendship.where(user_id: current_user.id, friend_id: params[:id])[0]
-    if friendship
-      friendship.destroy
+    friendship1 = Friendship.where(user_id: current_user.id, friend_id: params[:id])[0]
+    friendship2 = Friendship.where(user_id: params[:id], friend_id: current_user.id)[0]
+    if friendship1 && friendship2
+      friendship1.destroy
+      friendship2.destroy
```

Comment:
**[REQUIRED]**
* Everything connected with data processing should be encapsulated in models and used in controllers.
We can also have a method for destroying a friendship record in the friendship model that will handle the rejection of a friend request as we did in the confirm friend.
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright)

Reply:
I have implemented delete friendship, but reject friendship I have decided to leave it as it is because there is only one line method.